include/iconv.h.build.in

```diff
@@ -22,8 +22,20 @@
 
 #define _LIBICONV_VERSION 0x010F    /* version number: (major<<8) + minor */
 
+#ifdef LIBICONV_SHARED_BUILD
```

Comment:
CMake can generate this magic for you, see our other libraries.

Reply:
done